Gemfile

```diff
@@ -8,3 +8,4 @@ gem 'pry-byebug'
 gem 'rubocop'
 gem 'timecop'
 gem 'byebug'
+gem 'statsd-instrument'
```

Comment:
Since this is a production dependency, it needs to go in `kubernetes-deploy.gemspec` instead